app/manager-frontend/src/pages/UserHome.jsx

```diff
@@ -1,33 +1,34 @@
-import { Heading, SimpleGrid, VStack } from "@chakra-ui/react";
+import { Heading, SimpleGrid, VStack, Text } from "@chakra-ui/react";
 import { useUser } from "contexts/User";
 import { useTranslation } from "react-i18next";
+import { ActionCard } from "components/ActionCard";
+import { useSortingAndFiltering } from "helpers/hooks/useSortingAndFiltering";
+import { useActivitySourceList } from "api/activitysource";
+import { ActivitySourceSummary } from "components/ActivitySourceSummary";
+//import { useEffect } from "react";
 
 export const UserHome = () => {
   const { user } = useUser();
   const { t } = useTranslation();
+  const { data } = useActivitySourceList();
+
   return (
     <VStack align="stretch" px={8} w="100%" spacing={4} p={4}>
       <Heading as="h2" size="lg">
         {t("home.heading", { name: user?.fullName })}
       </Heading>
 
       <SimpleGrid minChildWidth="400px" spacing={4}>
-        {/* <ActionCard
-          href="process-overview"
-          icon={AiFillRead}
-          title="Proposals process overview"
-        >
-          <Text>
-            Review the process for submitting proposals using this system
-          </Text>
-        </ActionCard> */}
+        {data.map((item, index) => (
```

Comment:
:wrench: these cards shouldn't be in the `SimpleGrid`, they should be in the `VStack` (where the `Collections` header already was.

Also, there should be a header; probably `Activity Sources`